scanner/src/funTest/assets/analyzer-result_for_multiple_scanners.yml

```diff
@@ -0,0 +1,124 @@
+---
+repository:
```

Comment:
This just contains one package and one project. Would it make sense to create it programatically instead?

Reply:
Done. Could you please review the changes ? I think I over complicated it and don't need to build the dependency graph by hand.